README.md

```diff
@@ -196,6 +196,7 @@ Plugin configuration supports various options that are common and mode-specific
 |:------------|-------------|
 | **`validity`** | `string` validity of the issued certificate (default: `P30d`) |
 | **`labels`** | `[]string` list of GCP labels to apply to the certificate (format `k1=v1,k2=v2`) |
+| **`issuing_certificate_authority`** | `string` Optional. The resource ID of the CertificateAuthority that should issue the certificate. |
```

Comment:
Can we add:
"By default, the certificate will be issued from any of the active CAs in the CA Pool."